### PR TITLE
Improve Prioritization procedure's summarize sections

### PR DESCRIPTION
**Pull request by spastorino**
Explain a bit better the Prioritization summarize process

cc @rust-lang/wg-prioritization 

Please fix my english if it's broken :)

### CODE REVIEW

File: src/compiler/prioritization/procedure.md [hunk 1]
```diff
@@ -142,9 +142,11 @@ We will have checkins from @*WG-X* and @*WG-Y*
 #### Summarize stable/beta nominations
 
 - Add them to the agenda explaining:
+  - Who is the author of the PR
+  - Who is the assignee
+  - What issue fixes, it is a regression? what's the priority?
```
Comment (apiraino) [inline src/compiler/prioritization/procedure.md:147]:
```suggestion
  - Who the author of the PR is
  - Who the assignee is
  - Which issue fixes, it is a regression? what's the priority?
```

File: src/compiler/prioritization/procedure.md [hunk 2]
```diff
@@ -157,8 +159,10 @@ Try to follow the [General issues review process](#general-issues-review-process
 We should:
 
 - Add them to the agenda explaining:
-  - What are they waiting for
-  - Add important details
+  - Who is the author of the PR
+  - Who is the assignee
+  - What is the issue waiting for
+  - Add other important details
```
Comment (apiraino) [inline src/compiler/prioritization/procedure.md:165]:
```suggestion
  - Who the author of the PR is
  - Who the assignee is
  - What is the issue waiting for
  - Add other important details
```

File: src/compiler/prioritization/procedure.md [hunk 3]
```diff
@@ -186,10 +197,10 @@ We should:
 - Check if these issues were already discussed and in that case remove `I-nominated` label
 - Check if each issue is worth being discussed
 - Add them to the agenda explaining:
-  - Why was it nominated
-  - Who the assignee is
+  - Who is the assignee
```
Comment (apiraino) [inline src/compiler/prioritization/procedure.md:200]:
```suggestion
  - Who the assignee is
```

**Comment by spastorino** [comment]
Committed all @apiraino's suggestions. It should be ready to be merged now.

**Comment by XAMPPRocky** [comment]
Thank you for your PR!

**Pull request merged**